modules/migration-using-cpma.adoc

```diff
@@ -3,21 +3,26 @@
 [id='migration-using-cpma_{context}']
 = Using CPMA
 
+CPMA can run in remote mode, retrieving the configuration files using SSH, or in local mode, using local copies of the configuration files.
+
 .Prerequisites
 
-The following prerequisites apply to the {product-title} 3 cluster:
+The following prerequisites apply:
 
-* The cluster is updated to the latest synchronous release.
-* The environment health check has been run to confirm that there are no diagnostic errors or warnings.
-* The {product-title} version is 3.7, 3.9, 3.10, or 3.11.
+* The {product-title} 3 cluster must be updated to the latest synchronous release.
+* The environment health check must be run on the {product-title} 3 cluster to confirm that there are no diagnostic errors or warnings.
+* The {product-title} version must be 3.7, 3.9, 3.10, or 3.11.
+* If you are running CPMA in remote mode, the SSH user must have sudo permissions to read the configuration files. If this is not possible, you must copy the files locally and run CPMA in local mode.
```

Comment:
I think that SSH user details and Openshift user privileges below both belong to the .Procedure section as setup.

Reply:
What do you see as the difference between prerequisites and setup? 
SSH user details and ocp user privileges are more of a state than an action.

Reply:
One is the `Prerequisites` is about what's needed prior to do anything before downloading (or using) CPMA.

The second, `Setup` are the details required as part of getting CPMA working.